tasks/set_vars.yml

```diff
@@ -117,7 +117,7 @@
   set_fact:
     # We use escaped double quotes in the first regex_replace filter to be able
     # to add single quotes around app name in the query
-    apps_filter_query: "{{ apps_filter.split(',') | map('regex_replace', '(.*)', \"name=='\\1'\") | join(' || ') | regex_replace('(.*)', '[?\\1]') }}"
+    apps_filter_query: "[?{{ apps_filter.split(',') | map('regex_replace', '(\\w+)', \"name=='\\1'\") | join(' || ') }}]"
```

Comment:
And what about all the `json_query` made to create the vaults ? They still work ?

Reply:
Yup.